it.mijorus.whisper.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id" : "it.mijorus.whisper",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "whisper",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--device=all",
+        "--filesystem=xdg-run/pipewire-0"
```

Comment:
why do you need this one?

Reply:
`  "--socket=fallback-x11",`  was added automatically by Builder
` "--device=all",` I need to control audio devices
`  "--filesystem=xdg-run/pipewire-0"` same

Reply:
do you call pipewire directly or JACK?

Reply:
i call pipewire directly, with `pw link`